server/ntb/io/feed_parsers/ntb_events_api_xml.py

```diff
@@ -207,5 +313,23 @@ def _prefetch_vocabularies_items(self):
         # use qcode as a key to speed up work with it in the future methods
         self._vocabularies['eventoccurstatus'] = {s['qcode']: s for s in self._vocabularies['eventoccurstatus']}
 
+    def _prefetch_contacts(self):
+        """
+        Prefetch contacts `email` and `_id` from db,
+        """
+        self._contacts = {}
+
+        req = ParsedRequest()
+        req.projection = json.dumps({
+            'contact_email': 1
+        })
+        contacts = superdesk.get_resource_service(
+            'contacts'
+        ).get_from_mongo(req=req, lookup={})
```

Comment:
there is default pagination limit 200, not sure if that will be enough

Reply:
@petrjasek eve applies pagination during handling a http request.